app/dao/organisation_permissions_dao.py

```diff
@@ -0,0 +1,34 @@
+from app import db
+from app.models import OrganisationPermission
+
+# Default permissions for an organisation
+default_organisation_permissions = []
+
+
+def delete_organisation_permission(permission):
```

Comment:
I think you should be able to wrap this function with the `@autocommit` decorator than than manually doing `db.session.commit()` - at least this is what we do in a lot of the other dao files?

Maybe not for the `set_organisation_permission` one below though if we need that `_commit` arg.